src/app/components/Book/Book.jsx

```diff
@@ -1,29 +1,70 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import config from '../../../../appConfig.js';
+import { EReaderIcon, BookIcon } from 'dgx-svg-icons';
+import config from '../../../../appConfig';
 import utils from '../../utils/utils';
 
 const Book = ({ book }) => {
-  const fullImgSrc = (!book.imageSlug || book.imageSlug === '') ? `${config.baseUrl}src/client/images/book-place-holder.png` :
-    'https://contentcafe2.btol.com/ContentCafe/Jacket.aspx' +
-    `?&userID=NYPL49807&password=CC68707&Value=${book.imageSlug}&content=M&Return=1&Type=M`;
   const tagArray = book.tags.map(tag => tag.toLowerCase().split(' ').join('-'));
   const tagClasses = tagArray.join(' ');
 
+  const renderBookCoverImage = (imageUrl) => {
+    const defaultImageUrl = `${config.baseUrl}src/client/images/book-place-holder.png`;
+    const fullImgSrc = (!imageUrl || imageUrl === '') ? defaultImageUrl : imageUrl;
+    return (
+      <div className="book-item-image-box">
+        <img alt="" src={fullImgSrc} />
+      </div>
+    );
+  };
+
+  const renderTitle = (title, link) => {
+    const titleClass = 'book-item-title';
+
+    if (title && title !== '') {
+      if (link && link !== '') {
+        return <h4 className={titleClass}><a href={link}>{title}</a></h4>;
+      }
+      return <h4 className={titleClass}>{title}</h4>;
+    }
+    return null;
+  };
+
+  const renderAuthor = (author) =>
+    (author && author !== '') ? <p className="book-item-author">{author}</p> : null;
+
+  const renderCatalogLinks = (catalogUrl, ebookUrl) => {
+    const catalogLink = (catalogUrl && catalogUrl !== '') ?
+      <a href={catalogUrl} className="catalog-url">
+        <BookIcon width="32px" height="32px" ariaHidden />
+        <span>REQUEST THE BOOK</span>
+      </a> : null;
+    const ebookLink = (ebookUrl && ebookUrl !== '') ?
+      <a href={ebookUrl} className="ebook-url">
+        <EReaderIcon ariaHidden />
+        <span>REQUEST THE E-BOOK</span>
+      </a> : null;
+
+    return (catalogLink || ebookLink) ?
+      <div className="book-item-catalog-links">
+        {catalogLink}
+        {ebookLink}
+      </div> : null;
+  };
+
+  const renderDescription = (desc) =>
+    (desc && desc !== '') ? <p className="book-item-description">{desc}</p> : null;
+
   return (
     <li
-      className={`book-item nypl-column-half ${tagClasses}`}
+      className={`book-item ${tagClasses}`}
       key={book.title.trim()}
     >
-      <div className="book-item-image-box">
-        <img alt="" src={fullImgSrc} />
-      </div>
-      <div className="book-item-description">
-        <h4>
-          <a href="#">{book.title}</a>
-        </h4>
-        <p>{book.text}</p>
-      </div>
+      {renderBookCoverImage(book.imageUrl)}
+      {renderTitle(book.title, '#')}
```

Comment:
I thought we wanted to link the title to the `catalogUrl` as well?

Reply:
yea but for now since I don't want to do too much work on modifying the sample JSON, I just passed the hash. Once we merge Paul's latest JSON file, I can fix to pass book.encoreUrl or something...